src/api/app/models/bs_request.rb

```diff
@@ -273,19 +273,11 @@ def self.sourcediff_has_shown_attribute?(sourcediff)
   #               or :target_project
   # @return [Array]  Array of Request objects
   def self.with_open_reviews_for(attributes)
-    with_reviews(attributes).where(state: 'new').rewhere('bs_requests.state': 'review').map(&:bs_request)
+    Review.with_attributes(attributes).where(state: 'new').rewhere('bs_requests.state': 'review').map(&:bs_request)
   end
 
   def self.in_state_new(attributes)
```

Comment:
[nitpick] I would add a comment in that method similar to the one in [`self.with_open_reviews_for`](https://github.com/openSUSE/open-build-service/pull/6180/files#diff-d476b59ba5326de67e0c155e8e226d6fR274) to specify that this returns an Array. It is not evident/expected.